Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1976,6 +1979,7 @@ static bool isContextMenuEvent(const WebMouseEvent& event)
 Vector<PDFContextMenuItem> UnifiedPDFPlugin::scaleContextMenuItems() const
 {
     return {
+        { titleForContextMenuItemTag(ContextMenuItemTag::AutoSize) , m_documentLayout.shouldUpdateAutoSizeScale() == PDFDocumentLayout::ShouldUpdateAutoSizeScale::Yes, enumToUnderlyingType(ContextMenuItemTag::AutoSize), ContextMenuItemEnablement::Enabled, ContextMenuItemHasAction::Yes, ContextMenuItemIsSeparator::No },
```

Comment:
Pretty please teach `contextMenuItem` about this?

Reply:
Sure! I went ahead and modified the logic in `contextMenuItem` that was already there for determining the state to include this case. Let me know if you think if it needs to be modified in any way

Reply:
Looks good, thanks!